developer_tests/run_tests.csh

```diff
@@ -209,6 +214,12 @@ set FAILURE = 0
 
 ( ./run_tests.csh > ${LOGDIR}/location_tests.out ) || set FAILURE = 1
 
+if ( $FAILURE ) then
+  echo "------------------------------------------------------------------"
+  echo "ERROR - unsuccessful location tests at "`date`
+  echo "------------------------------------------------------------------"
+endif
+
```

Comment:
I am not sure this adds anything.  Do you want to count the failures or something?

Reply:
reverted this to the master branch ( no extra prints)